barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/mega_flavor.hpp

```diff
@@ -416,9 +416,6 @@ class MegaFlavor {
                                                                              this->circuit_size,
                                                                              this->pub_inputs_offset);
             relation_parameters.public_input_delta = public_input_delta;
-            auto lookup_grand_product_delta = compute_lookup_grand_product_delta(
-                relation_parameters.beta, relation_parameters.gamma, this->circuit_size);
-            relation_parameters.lookup_grand_product_delta = lookup_grand_product_delta;
 
             // Compute permutation and lookup grand product polynomials
             compute_grand_products<MegaFlavor>(this->polynomials, relation_parameters);
```

Comment:
this flavor should delete the to_field_elements function as its in flavor.hpp now.

Reply:
done